### DISCUSSION TITLE
Elsa as a .NET Aspire integration

### DISCUSSION THREAD

**Discussion by svp95bos** [elsa-workflows/elsa-core#4671] [Ideas]
Microsoft has just released their first version of the [.NET Aspire platform](https://github.com/dotnet/aspire). It would be great to see Elsa become available as a [.NET Aspire integration](https://learn.microsoft.com/en-us/dotnet/aspire/fundamentals/integrations-overview) to be able to use it in this context.

**Comment by sfmskywalker** [comment]
Thanks for the suggestion. Elsa now uses .NET Aspire in the [Elsa Package Catalog AppHost](https://github.com/elsa-workflows/elsa-package-catalog/tree/main/src/Elsa.Catalog.AppHost): it references `Aspire.AppHost.Sdk` and composes the catalog API and Azure resources. That is an application-specific host, not a reusable Elsa integration package.

I could not find a general Aspire integration package or an active implementation plan in the Elsa Core, Extensions, Apps, Samples, or Guides repositories. If a generic integration is still needed, please open a focused request describing the expected resource model, hosting/deployment target, and which Elsa capabilities should be exposed through Aspire.

As this original idea has been inactive and no concrete package proposal is currently defined, I’m closing the discussion.
